### PR TITLE
Diagnostic, Mar. 28

### PR DESCRIPTION
**Pull request by susandaiute**
Answer functions ins & outs questions.  Unfinished.

### CODE REVIEW

File: lib/diagnostic.js [hunk 1]
```diff
@@ -4,15 +4,25 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min() {
-
+const min = function min(array) {
+let array = [];
```
Comment (RDegnen) [inline lib/diagnostic.js:8]: your already passing in `array` as a parameter

File: lib/diagnostic.js [hunk 2]
```diff
@@ -4,15 +4,25 @@
...
+let x = array[i];
```
Comment (RDegnen) [inline lib/diagnostic.js:9]: theres no need to define x as `array[i]` first, i isn't defined here. And `array[i]` should only be used inside the loop where i would be defined, or in your case `x`

File: lib/diagnostic.js [hunk 3]
```diff
@@ -4,15 +4,25 @@
...
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-const sum = function sum() {
+const sum = function sum(array) {
+  let array = [];
+  let x = array[i];
+  for (x=0;x<array.length;x++){
+    return x+sum;
+  }
```
Comment (RDegnen) [inline lib/diagnostic.js:25]: same as above

**Comment by RDegnen** [comment]
Do you need a review of for loops?

**Comment by susandaiute** [comment]
Yup!

**Pull request closed without merging**